app/src/main/java/at/roteskreuz/stopcorona/model/entities/session/DbSession.kt

```diff
@@ -12,10 +14,19 @@ import at.roteskreuz.stopcorona.skeleton.core.model.entities.DbEntity
 )
 data class DbSession(
     @PrimaryKey()
-    val token: String = "",
-    val warningType: WarningType
+    var currentToken: String = "",
+    val warningType: WarningType,
+    var processingPhase: ProcessingPhase,
+    var yellowDay: ZonedDateTime?
```

Comment:
do you mean the first yellow day? or should it be a timespan? can you rename the variable to make it's use more obvious?

Reply:
fixed